src/TurbulenceClosures/turbulence_closure_implementations/anisotropic_biharmonic_diffusivity.jl

```diff
@@ -16,15 +16,15 @@ end
 """
     AnisotropicBiharmonicDiffusivity(; νh, νv, κh, κv)
 
-Returns parameters for a fourth-order, anisotropic biharmonic diffusivity closure with 
-constant horizontal and vertical biharmonic viscosities `νh`, `νv` and constant horizontal 
-and vertical tracer biharmonic diffusivities `κh`, `κv`. 
+Returns parameters for a fourth-order, anisotropic biharmonic diffusivity closure with
+constant horizontal and vertical biharmonic viscosities `νh`, `νv` and constant horizontal
+and vertical tracer biharmonic diffusivities `κh`, `κv`.
 `κh` and `κv` may be `NamedTuple`s with fields corresponding
 to each tracer, or a single number to be a applied to all tracers.
 The tracer flux divergence associated with an anisotropic biharmonic diffusivity is, for example
 
 ```math
-    ∂ᵢ κᵢⱼ ∂ⱼc = (κh ∇h⁴ + κv ∂z⁴) c
+    ∂ᵢ κᵢⱼ ∂ⱼc = (κh ∇⁴h + κv ∂⁴z) c
```

Comment:
It's really a notational point, but the reason I chose the representation is because the notion expressed is "vertical partial derivative to the fourth power" or "horizontal Laplacian to the fourth power". I felt putting the 4 superscript _outside_ the `∇h` was the best way to express this notion. Was this logic considered and rejected? It seems ambiguous because in true mathematical notation both the `h` and `z` would be subscripts.

Reply:
The reason I changed it was because I found `∇h⁴ᶜᶜᵃ` a bit harder to visually parse the superscripts than `∇⁴hᶜᶜᵃ`. By separating the superscripts it's easier to read off the order of the derivative and the location of the result.

I agree that "vertical partial derivative to the fourth power" is better than "fourth partial derivative along z". The `h` and `z` should be subscripts but unfortunately Unicode doesn't give us those :(

I think the meaning of `∇⁴hᶜᶜᵃ` and `∂⁴zᵃᵃᶠ` is still pretty clear but I'm okay with either way. If you'd prefer `∇h⁴ᶜᶜᵃ` and `∂z⁴ᵃᵃᶠ` I can switch them back.

Reply:
That's a good reason; I'm fine to keep your change.